datastore/data-loader/Dockerfile

```diff
@@ -29,7 +30,16 @@ RUN python -m grpc_tools.protoc  \
     --grpc_python_out="${DOCKER_PATH}"
 
 COPY "./parameters.py" "${DOCKER_PATH}/parameters.py"
+
+# Copy both loaders and conditionally use the wanted one
+COPY "./client_knmi_station_ingest.py" "${DOCKER_PATH}/client_knmi_station_ingest.py"
 COPY "./client_knmi_station.py" "${DOCKER_PATH}/client_knmi_station.py"
 
+RUN if [ "${THROUGH_INGEST}" = "true" ]; then \
+      mv "${DOCKER_PATH}/client_knmi_station_ingest.py" "${DOCKER_PATH}/client_knmi_station.py"; \
+    else \
+      rm "${DOCKER_PATH}/client_knmi_station_ingest.py"; \
+    fi
+
```

Comment:
A bit nasty, but I don't see a better way.

Reply:
Yeah, it is either this or two separate `Dockerfiles` to be maintained. It is ugly, but easier to main and this way we have have a bit better `layer` caching when both of the files are built in same job.